src/components/Choice/src/ChoiceOption.vue

```diff
@@ -4,15 +4,28 @@
 			$s.Button,
 			{ [$s.selected]: isSelected },
 		]"
+		:style="style"
 		@click="selectSelf"
 	>
 		<slot />
 	</button>
 </template>
 
 <script>
+import chroma from 'chroma-js';
 import key from './key';
 
+function getContrast(chromaBg, targetChromaFg) {
```

Comment:
also, since all this logic does is generate some CSS variables I think we should move it into the parent Choice component instead of repeating this work in every ChoiceOption component (since it always evaluates to the same result anyway)

Reply:
@mattmewton Just made `getContrast` a util